tests/packagedcode/data/rubygems/gemspec/arel.gemspec.expected.json

```diff
@@ -1,187 +1,62 @@
-{
-  "platform": "ruby", 
-  "name": "arel", 
-  "version": "2.0.7.beta.20110429111451", 
-  "homepage": "http://github.com/rails/arel", 
-  "summary": "Arel is a SQL AST manager for Ruby", 
-  "description": "Arel is a SQL AST manager for Ruby. It\n\n1. Simplifies the generation complex of SQL queries\n2. Adapts to various RDBMS systems\n\nIt is intended to be a framework framework; that is, you can build your own ORM\nwith it, focusing on innovative object and collection modeling as opposed to\ndatabase compatibility and query generation.", 
-  "licenses": null, 
-  "email": [
-    "aaron@tenderlovemaking.com", 
-    "bryan@brynary.com", 
-    "miloops@gmail.com", 
-    "nick@example.org"
-  ], 
-  "authors": [
-    "Aaron Patterson", 
-    "Bryan Halmkamp", 
-    "Emilio Tagua", 
-    "Nick Kallen"
-  ], 
-  "date": "2011-04-29 00:00:00 UTC", 
-  "requirements": null, 
-  "dependencies": [
-    "minitest (>= 2.0.2, development)", 
-    "hoe (>= 2.9.1, development)"
-  ], 
-  "files": [
```

Comment:
Do we still capture these and test files below?

Reply:
No not for now